packages/elements-dev-portal/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@stoplight/elements-dev-portal",
+  "version": "1.0.0-alpha.1",
+  "description": "UI components for composing beautiful developer documentation.",
+  "keywords": [],
+  "sideEffects": [
+    "web-components.min.js",
+    "src/web-components/**"
+  ],
+  "homepage": "https://github.com/stoplightio/elements",
+  "bugs": "https://github.com/stoplightio/elements/issues",
+  "author": "Stoplight <support@stoplight.io>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stoplightio/elements"
+  },
+  "license": "Apache-2.0",
+  "main": "",
+  "files": [
+    "**/*"
+  ],
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "build": "yarn build.react && yarn build.webcomponents",
+    "build.react": "sl-scripts build",
```

Comment:
There is a `build.styles` that runs `build-styles.js` during postbuild in elements package. Don't we need that here too?

Reply:
I haven't moved any styles yet, I plan to do it in a separate PR.

After this PR folks still need to import styles from `@stoplight/elements`.